dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/AbstractCrudController.java

```diff
@@ -290,23 +290,30 @@
             setUserContext( null, new TranslateParams( false ) );
         }
 
-        if ( !aclService.canRead( user, getEntityClass() ) )
+        try
         {
-            throw new ReadAccessDeniedException( "You don't have the proper permissions to read objects of this type." );
-        }
+            if ( !aclService.canRead( user, getEntityClass() ) )
+            {
+                throw new ReadAccessDeniedException( "You don't have the proper permissions to read objects of this type." );
+            }
 
-        List<String> fields = Lists.newArrayList( contextService.getParameterValues( "fields" ) );
+            List<String> fields = Lists.newArrayList( contextService.getParameterValues( "fields" ) );
 
-        if ( fields.isEmpty() )
-        {
-            fields.add( ":all" );
-        }
+            if ( fields.isEmpty() )
+            {
+                fields.add( ":all" );
+            }
```

Comment:
Is the extra indentation here correct?

Reply:
```java
        try
        {
            if ( !aclService.canRead( user, getEntityClass() ) )
            {
                throw new ReadAccessDeniedException( "You don't have the proper permissions to read objects of this type." );
            }

            List<String> fields = Lists.newArrayList( contextService.getParameterValues( "fields" ) );

            if ( fields.isEmpty() )
            {
                fields.add( ":all" );
            }
```
When looking at the complete code, the indentation in line 302 to 305 is correct. The if statement has an indentation of 12 space characters and the method invocation has an indentation of 16 space characters.